src/solvers/solver_hardness.cpp

```diff
@@ -91,10 +93,20 @@ void solver_hardnesst::register_clause(const bvt &bv)
 {
   current_hardness.clauses++;
   current_hardness.literals += bv.size();
+  std::vector<int> clause;
+
   for(const auto &literal : bv)
   {
     current_hardness.variables.insert(literal.var_no());
+
+    int signed_literal = literal.var_no();
+    if(literal.sign())
+      signed_literal = -signed_literal;
+    clause.push_back(signed_literal);
   }
+  clause.push_back(0);
+  std::sort(clause.begin(), clause.end());
```

Comment:
Do we want to sort before or after we add the `0`?

Reply:
Post processing of the SAT core cnf runs sort for the entire clause, including the zero. So might as well sort it here too.